tests/scalers_go/custom_hpa_name/custom_hpa_name_test.go

```diff
@@ -0,0 +1,158 @@
+//go:build e2e
+// +build e2e
+
+package custom_hpa_name
+
+import (
+	"fmt"
+	"github.com/joho/godotenv"
+	. "github.com/kedacore/keda/v2/tests"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"testing"
```

Comment:
Could you please split imports into 3 blocks, as it is done here: https://github.com/kedacore/keda/blob/2b3fdc0e4d0c6283524be1110ef793934efee3fa/tests/scalers_go/azure_keyvault_queue/azure_keyvault_queue_test.go#L7-L23

@v-shenoy any idea why this hasn't been catched by the linter?

Reply:
I think it's because of the build tag. 

https://golangci-lint.run/usage/configuration/ 
Think we need to add

```yaml
run:
  build-tags:
  - e2e
```

Reply:
I have opened [a PR](https://github.com/kedacore/keda/pull/3189) adding this, thanks for the tip 🙇